src/net_processing.cpp

```diff
@@ -1301,7 +1316,12 @@ void PeerManagerImpl::UpdateBlockAvailability(NodeId nodeid, const uint256 &hash
     }
 }
 
-void PeerManagerImpl::FindNextBlocksToDownload(const Peer& peer, unsigned int count, std::vector<const CBlockIndex*>& vBlocks, NodeId& nodeStaller)
+void PeerManagerImpl::FindNextBlocksToDownload(
```

Comment:
In 058fecdc7d2c99af9ae6aedc68c3600321e7c565: just a suggestion but I'd change `FindNextBlocksToDownload` to return `std::vector<const CBlockIndex*>` instead of passing it in the parameters.